### ISSUE TITLE
Type of computing

### ISSUE THREAD

**Issue by kavkapet** [storm-fsv-cvut/smoderp2d#171]
Compute only surface runoff (sheet and rill) in case empty inpute stream data

**Comment by jerabekjak** [comment]
Which one is working now? Numbers of characters?

![Screenshot from 2023-10-16 16-42-02](https://github.com/storm-fsv-cvut/smoderp2d/assets/10115010/189787e8-0ba6-49f2-87c8-79dfe98c2c71)

**Comment by landam** [comment]
> @landam @pesekon2 Which one is working now? Numbers of characters?

@jerabekjak We switched to strings in June: https://github.com/storm-fsv-cvut/smoderp2d/commit/4c5e18f230fad83252c5f989707543c89100551d#diff-919ea52275af6d3e986f274bdf5eef6352a77d5d99b6b85999ff4fd1fa7ec04a

**Comment by pesekon2** [comment]
@jerabekjak: The right one, using words. I have no idea why there are numbers in `gistest`, probably some ancient remnants. However, I have tried to change the value in `gistest.ini` and have found that it is probably not used at all. Even if I delete the `.ini` file, the tests pass. @landam: Is that correct? Isn't that wrong?

**Comment by landam** [comment]
@pesekon2 You are right, `gistest.ini` is not currently used. But it's planned to be used by `test_002` (use dpre from `test_001` + cmd provider): https://github.com/storm-fsv-cvut/smoderp2d/blob/master/tests/test_grass.py#L26

**Comment by landam** [comment]
@jerabekjak @kavkapet Can we close this issue?
